content/influxdb/v2.1/write-data/no-code/use-telegraf/dual-write.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Dual write to InfluxDB OSS and InfluxDB Cloud
+description: Write data to both OSS and Cloud simultaneously.
+menu:
+  influxdb_2_1:
+    parent: Telegraf (agent)
+weight: 201
+---
+
+Write to both InfluxDB OSS and InfluxDB Cloud with Telegraf.
+
+You might want to set up dual write if you want a backup of your data in two places, or if you're migrating from OSS to Cloud.
+
+The sample configuration below uses the [InfluxDB v2 output plugin](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/influxdb_v2) twice: first pointing to the OSS instance and then to the cloud instance.
+
+## Sample configuration
+
+```
+[[inputs.cpu]]
+[[inputs.mem]]
+
+## Any other inputs, processors, aggregators that you want to include into your config.
+## Without any namepass, tagpass filters, or settings changes, your data should get identically written to the two outputs below.
```

Comment:
Can probably move this line out of the sample config actually and into the description. Can even change it to "Without any [metric filtering](https://docs.influxdata.com/telegraf/latest/administration/configuration/#metric-filtering) or [setting changes](https://docs.influxdata.com/telegraf/v1.20/administration/configuration/#output-configuration) to your output plugins, your data should get identically written to your OSS and Cloud instances"